src/ceph.in

```diff
@@ -715,7 +714,7 @@ def main():
             timeout = 5
 
         hdr('Monitor commands:')
-        print('[Contacting monitor, timeout after %d seconds]' % timeout)
+        print('[Contacting monitor, timeout after %d seconds]' % timeout, file=sys.stderr)
```

Comment:
why would we print this info to stderr? it's but informational string, not error message.